data_generalization/date_normalizer.py

```diff
@@ -0,0 +1,94 @@
+"""
+Run this script from the root folder like this: python -m data_generalization.date_normalizer
+"""
+
+from root import DATA_DIR, DATE_NORMALIZATION_DIR
+from pathlib import Path
+from shutil import copyfile
+import pandas as pd
+from tqdm import tqdm
+import re
+
+
+# dict containing all the months in the languages german, french and italian
+months = {
+    'de': ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember'],
+    'fr': ['janvier', 'février', 'mars', 'avril', 'mai', 'juin', 'juillet', 'août', 'septembre', 'octobre', 'novembre', 'décembre'],
+    'it': ['gennaio', 'febbraio', 'marzo', 'aprile', 'maggio', 'giugno', 'luglio', 'agosto', 'settembre', 'ottobre', 'novembre', 'dicembre']
+}
+
+# dict containing the word "year" in the languages german, french and italian
+year_word = {
+    'de': 'Jahr',
+    'fr': 'année',
+    'it': 'anno'
+}
+
+# dict containing the word "month" in the languages german, french and italian
+month_word = {
+    'de': 'Monat',
+    'fr': 'mois',
+    'it': 'mese'
+}
+
+# dict containing the word "date" in the languages german, french and italian
+date_word = {
+    'de': 'Datum',
+    'fr': 'date',
+    'it': 'data'
+}
```

Comment:
I guess this should be renamed to "day", etc. to be consistent with Month and Year, right?

Reply:
No, actually not. 
These words are there to replace datums as a whole with one word. ("18.05.2014" --> "date")
Since sometimes months or years occur separately, there are separate words for them too.
Single day specifications (like 14.) on the other hand hardly occur alone, and are also more difficult to catch, which is why there are no replacement words for them.

Reply:
Aah, I see. Thanks for the clarification.